### ISSUE TITLE
Installed Deliver Skeleton and Admin, now errors

### ISSUE THREAD

**Issue by 2braincells2go** [getgrav/grav-plugin-admin#695]
Hi. First time user of Grav and trying out as possible blogging platform for school. Really like and excellent since we cannot use any database (mySQL, etc).

I first installed the rc core with admin, no problems at all. Decided to try the Deliver skeleton on same machine (which is a VPS centro) and went downhill:)

All is setup and working, but getting this error in red at top of all admin pages:

` You are running Grav v1.0.10. You must update to the latest Grav v1.1.x release in order to ensure compatibility. This may require switching to Testing GPM releases in the System configuration.`

I do not see any "Testing GPM releases" in the system config?

Also at very bottom of admin dashboard / configuration / system is this jumble of characters:

` ":"gt","<":"lt"}" />`

Was hoping to simple to use/install as sometime younger kids will be using:) If every install requires command line work, maybe not for the younger user...

Any input greatly appreciated.

Woody

**Comment by rhukster** [comment]
This is caused by you copying over an existing 1.1 install of Grav with a skeleton containing Grav 1.0.10 (probably)

Skeletons don't usually come with admin plugin but it appears you have the 1.1RC admin installed with the 1.0.10 version of Grav. 

As the 1.1 version of Grav is required by the 1.1 version of the admin you are getting this message. However the RC version is not released yet so it's not available to update from 1.0.10. The testing option was added in 1.1 so you won't be able to upgrade until it's released as stable. 

Easy fix is to download Grav 1.1RC core package and copy that over your current install. Alternatively you can wait a few days as we are about to release 1.1 as stable. Then you will be able to just hit the update option in the admin.

**Comment by rhukster** [comment]
Btw feel free to reopen this if your issue is not resolved :)

**Comment by 2braincells2go** [comment]
Thanks for info Andy, good to know.

But, did not install over other, brand new directory.

What I did do was install the deliver skeleton and then copy the admin from https://github.com/getgrav/grav/releases/download/1.0.10/grav-admin-v1.0.10.zip

Let me back up and make sure I did not space out, who knows, getting old, lol...

Thanks again.

**Comment by rhukster** [comment]
Ok cool!

**Comment by 2braincells2go** [comment]
FYI:

"Easy fix is to download Grav 1.1RC core package and copy that over your current install"

Fixed issue!

Thank you much Andy.

**Comment by 2braincells2go** [comment]
Not sure what the heck went wrong Andy. Now I am getting the message when clicking on any header links (deliver skeleton) except home/blog which work:

```
RuntimeException (400)
Template "modular_alt.html.twig" is not defined.
```

After few hours of reading docs and forum, at a loss of solution...

**Comment by rhukster** [comment]
It just means you have a page that is looking for the `modular_alt` Twig template. Clearly this template does not exist in your theme (or a plugin) and Grav can't find it.  

By default Grav uses the first part of the markdown page to determine what template to use, ie. `blog.md` will look for `blog.html.twig` in the template to render itself.  However you can override this with `template: modular_alt` in your page's frontmatter.  This could be happening.

Or the skeleton might have some modular stuff going on and you renamed something that was `modular` to `modular_alt`.  `modular.html.twig` is a pretty standard name for a page that renders modular content, but it could of been `modular_alt` and you might of removed the `modular_alt.html.twig` file from the theme.

Really there's lots of ways this might of broken accidentally ;)

**Comment by 2braincells2go** [comment]
Thank you for input. Looks like I have a lot to study:)

Think I will hold off and wait for rc release to become a stable release. Any idea when this may happen?

Really, really like grav and think school could get a lot of mileage out of it, but got to make sure the kids (or better terms - younger adults) can use without easily messing up things.

Cheers!

**Comment by rhukster** [comment]
RC -> Stable is imminent!   Today or tomorrow!

**Comment by 2braincells2go** [comment]
FYI: I deleted everything and started over. Great way to learn, lol.

Added the Striped theme.

Soon as I added the themer (https://github.com/sommerregen/grav-plugin-themer) plugin, things broke and started getting the runtime exception errors again. Deactivated and things back to normal.

Not expecting solution, just letting you know.

Looking forward to release of new version. Have two days to decide blogging platform so IT personnel can approve for new school year:) Glad I found Grav in time!
